lib/gui/app/pages/main/controllers/drive-selection.js

```diff
@@ -42,9 +43,35 @@ module.exports = function (DriveSelectorService) {
       return _.head(drives).description || 'Untitled Device'
     }
 
+    // eslint-disable-next-line no-magic-numbers
+    if (drives.length === 0) {
+      return 'No targets found'
+    }
+
     return `${drives.length} Devices`
   }
 
+  /**
+   * @summary Get drive subtitle
+   * @function
+   * @public
+   *
+   * @returns {String} - drives subtitle
+   *
+   * @example
+   * console.log(DriveSelectionController.getDrivesSubtitle())
+   * > '32 GB'
+   */
+  this.getDrivesSubtitle = () => {
+    const drive = selectionState.getCurrentDrive()
+
+    if (drive) {
+      return prettyBytes(drive.size)
+    }
+
+    return 'Please plug at least one target device'
```

Comment:
Missing a word here, I believe: "plug" -> "plug in"

Reply:
Yeah, I wonder if 'insert' would be better?

Reply:
Yeah, I think we should go with "insert"